_data/projects/angular2_universal.yml

```diff
@@ -0,0 +1,12 @@
+name: Universal Angular 2
+desc: Universal (isomorphic) javascript support for Angular 2
+site: https://github.com/angular/universal
+tags:
+- JavaScript
+- web
+- Angular 2
+- Universal JavaScript
+- Isomorphic JavaScript
+upforgrabs:
+  name: hotlist: community-help
```

Comment:
The `:` here is causing a parsing issue. You'll need to wrap this value in double-quotes.